### PR TITLE
add a simple input handler

### PR DESCRIPTION
**Pull request by FredTheDino**
inputs! :D

### CODE REVIEW

File: lingon/src/input.rs [hunk 1]
```diff
@@ -0,0 +1,201 @@
...
+                    ..
+                } => {
+                    (Device::Quit, KeyState::Down(frame))
+                }
+                Event::KeyDown {
+                    keycode,
+                    ..
+                } => {
+                    if keycode.is_some() {
+                        (Device::Key(keycode.unwrap()), KeyState::Down(frame))
```
Comment (sornas) [inline lingon/src/input.rs:136]:
```suggestion
                    if let Some(keycode) {
                        (Device::Key(keycode), KeyState::Down(frame))
```

File: lingon/src/input.rs [hunk 2]
```diff
@@ -0,0 +1,201 @@
...
+                    } else {
+                        continue;
+                    }
+                }
+                Event::KeyUp {
+                    keycode,
+                    ..
+                } => {
+                    if keycode.is_some() {
+                        (Device::Key(keycode.unwrap()), KeyState::Up(frame))
```
Comment (sornas) [inline lingon/src/input.rs:146]: dito

**Review by sornas — COMMENTED** [review]
In this PR the windows ignores when I press shift+mod+c in my WM which previously closed the window. Pressing escape works.

**Comment by FredTheDino** [comment]
> In this PR the windows ignores when I press shift+mod+c in my WM which previously closed the window. Pressing escape works.

I did dumdum, should be fixed! :)

**Review by sornas — APPROVED** [review]
:+1: works great

**Pull request merged**